stage.py

```diff
@@ -589,6 +589,17 @@ def text(self, text, hightlight=False):
                 if self.row >= self.height:
                     self.row = 0
 
+    def measure(self, text, hightlight=False):
+        """Measure the display size of text in pixels."""
+        rows = 1
+        cols = 0
+        for c in text:
+            if ord(c) >= 32:
+                cols += 1
```

Comment:
This will keep incrementing ``cols`` for every line, so if you have 3 lines of text with 10 characters each, you will end up with ``cols=30``. What you want is to reset ``cols`` to ``0`` on each line as ``text()`` does, and keep a maximum of all lines.